### PR TITLE
UML 380: Added feature test files for the actor account creation scenarios

### PR DESCRIPTION
**Pull request by SeemaMenon**

### CODE REVIEW

File: service-front/app/features/actor-account-creation.feature [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    Then I receive unique instructions on how to activate my account
+
+  @ui @integration
+  Scenario Outline: As a new user I want to be shown the mistakes I make while creating an account
+    Given I am not a user of the lpa application
+    And I want to create a new account
+    When I have not provided required information for account creation such as <email1> <email2> <password1> <password2> <terms>
+    Then I should be told my account could not be created due to <reasons>
+    Examples:
+      | email1 | email2 | password1 | password2 | terms | reasons                          |
```
Comment (cooperaj) [inline service-front/app/features/actor-account-creation.feature]: These look great now. My only comment would be to make the conditions match more exactly. For instance for "Confirm you password" I would expect that all fields _except_ password2 be filled with correct data. That way we're making clear we're testing the exact thing we're after.

File: service-front/app/features/actor-account-creation.feature [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  @ui @integration
+  Scenario Outline: As a new user I want to be shown the mistakes I make while creating an account
+    Given I am not a user of the lpa application
+    And I want to create a new account
+    When Creating account I provide mismatching <password> <confirm_password>
+    Then I should be told my account could not be created due to <reasons>
+    Examples:
+      | password       | confirm_password  |reasons                     |
+      | password       | pass              | The passwords did not match|
+      | password       | pass              | Your password must include at least one capital letter (A-Z)|
```
Comment (cooperaj) [inline service-front/app/features/actor-account-creation.feature]: Here for instance I'd set confirm_password to be "password". We'd still get the capital letter error but it's more explicit.

File: service-front/app/features/context/UI/AccountContext.php [hunk 3]
```diff
@@ -312,4 +493,83 @@ public function iAmGivenInstructionOnHowToChangeDonorOrAttorneyDetails()
         $this->assertPageContainsText('Let us know if a donor\'s or attorney\'s details change');
         $this->assertPageContainsText('Find out more');
     }
+
+    /**
+     * @When /^I have not provided required information for account creation such as (.*)(.*)(.*)(.*)(.*)$/
```
Comment (cooperaj) [inline service-front/app/features/context/UI/AccountContext.php]: It's possible to use named parameters for this. I'd look into that as it's a little more clear than (.*). I *think* its something like:

`@When I have not provided required information for account creation such as :email1 :email2 :password1 :password2 :terms`

Though double check that.

Comment (cooperaj) [reply service-front/app/features/context/UI/AccountContext.php]: Notice its no longer a regular expression.

File: service-front/app/features/actor-account-creation.feature (hunk 1 above)
Comment (SeemaMenon) [reply service-front/app/features/actor-account-creation.feature]: Ok, done.

File: service-front/app/features/actor-account-creation.feature (hunk 2 above)
Comment (SeemaMenon) [reply service-front/app/features/actor-account-creation.feature]: Done

File: service-front/app/features/context/UI/AccountContext.php (hunk 3 above)
Comment (SeemaMenon) [reply service-front/app/features/context/UI/AccountContext.php]: Not able to get that working now. I doubt the named parameters working for table the way we thought above, may be I am wrong..but parking this for now to proceed with current ticket.

**Pull request merged**